composer.json

```diff
@@ -15,20 +15,23 @@
         }
     ],
     "require": {
-        "php": "^7.3 || ^8.0",
+        "php": "^7.2 || ^8.0",
         "ext-pdo": "*",
         "doctrine/dbal": "^2.11 || ^3.0",
+        "doctrine/cache": "^1.12",
```

Comment:
i am a bit unhappy with this, i'd prefer to keep being able to install jackalope along with doctrine/cache 2 - otherwise we create problematic combinations of requirements.

how is the problem related to PHP 7.2? or is it a general problem of the jackalope-doctrine-dbal 1.x branch?

Reply:
ah ok, just found that doctrine/cache 2 only has interfaces but not the implementations. and the 1.x branch fails. lets do this then.